hack/verify.sh

```diff
@@ -50,14 +50,19 @@ fi
 # Checks the Terraform version used by the module, download the Terraform binary
 # for that version
 if grep "required_version.*0.12.*" "${REPO_ROOT}/main.tf"; then
-    TERRAFORM_VERSION="0.12.4"
+    TERRAFORM_VERSION="0.12.20"
```

Comment:
Could we bump this? I think v0.12.24 is out now.

Reply:
Done! Also updated the mention in `example/README.md` too.